Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1624,6 +1624,38 @@ static BOOL annotationIsLinkWithDestination(PDFAnnotation *annotation)
 
     PDFElementTypes pdfElementTypes { PDFElementType::Page };
 
+#if HAVE(PDFPAGE_AREA_OF_INTEREST_AT_POINT)
+    if ([page respondsToSelector:@selector(areaOfInterestAtPoint:)]) {
+        PDFAreaOfInterest areaOfInterest = [page areaOfInterestAtPoint:pointInPDFPageSpace];
+
+        if (areaOfInterest & kPDFTextArea)
+            pdfElementTypes.add(PDFElementType::Text);
+
+        if (areaOfInterest & kPDFAnnotationArea)
+            pdfElementTypes.add(PDFElementType::Annotation);
+
+        if (areaOfInterest & kPDFLinkArea)
+            pdfElementTypes.add(PDFElementType::Link);
```

Comment:
It seems like this loses the "with destination" part of annotationIsLinkWithDestination

Reply:
Good catch, adding it back.

Reply:
Fixed in [d5a5e06](https://github.com/WebKit/WebKit/commit/d5a5e065cd9d63be601cfaa59b95e6c2426a7555)